libsql-server/src/lib.rs

```diff
@@ -632,62 +641,74 @@ where
         Arc<dyn Fn() -> InnerWalManager + Sync + Send + 'static>,
```

Comment:
```suggestion
        Arc<dyn Fn(ns: Option<NamespaceName>) -> InnerWalManager + Sync + Send + 'static>,
```

I have a question for the follow up PR. I need to change the definition of the `make_wal_manager` to pass Namespace to the `DurableWALManager`. I am wondering what would be the right place to do it?

Reply:
that's not going to work, unfortunately, you can assume the namespace name at this level, but only when the db is opened. Consider what happens when you run `ATTACH otherdb`.

Reply:
IOW, you only know what db is opened when `WalManager::open` is called

Reply:
yeah, see how I did it below. I'm not a fan of doing it this way, but I haven't come up with a better alternative either.